content/docs/reference-react-dom.md

```diff
@@ -62,15 +62,15 @@ If the optional callback is provided, it will be executed after the component is
 ReactDOM.hydrate(element, container[, callback])
 ```
 
-Same as [`render()`](#render), but is used to hydrate a container whose HTML contents were rendered by [`ReactDOMServer`](/docs/react-dom-server.html). React will attempt to attach event listeners to the existing markup.
+To samo, co [`render ()`](#render), ale służy do uwadniania kontenera, którego treść HTML została wyrenderowana przez [`ReactDOMServer`](/docs/react-dom-server.html). React podejmie próbę dołączenia detektorów zdarzeń do istniejących znaczników.
```

Comment:
```suggestion
Działa podobnie do funkcji [`render()`](#render), ale służy do inicjalizowania danymi kontenera, którego zawartość HTML została wyrenderowana przez [`ReactDOMServer`](/docs/react-dom-server.html). React podejmie próbę dołączenia detektorów zdarzeń do istniejących elementów.
```
"Detektor zdarzeń" brzmi całkiem nieźle. Być może łatwiej by się go używało niż "obserwatora zdarzeń", który odmienia się w nienaturalny sposób (niemęskos.). Co na to reszta? @marcinostrouch @bartlomiejzuber @TeoTN @iamarek @cvqprs?